Pod/Classes/TTRangeSlider.m

```diff
@@ -223,8 +224,13 @@ - (void)updateLabelValues {
     NSNumberFormatter *formatter = (self.numberFormatterOverride != nil) ? self.numberFormatterOverride : self.decimalNumberFormatter;
 
     self.minLabel.string = [formatter stringFromNumber:@(self.selectedMinimum)];
-    self.maxLabel.string = [formatter stringFromNumber:@(self.selectedMaximum)];
-    
+  
+    if (self.allowInfiniteSelectedMax && [NSNumber numberWithFloat:self.selectedMaximum] == [NSNumber numberWithFloat:self.maxValue] ) {
```

Comment:
Do you want to make this conditional logic a function? You dont have to just thought it might be useful later.

Side note: What did the problem end up being that caused this to be an infinite loop and freeze the app?